src/node/txdownloadman_impl.h

```diff
@@ -194,6 +194,12 @@ class TxDownloadManagerImpl {
     /** Helper for getting deduplicated vector of Txids in vin. */
     std::vector<Txid> GetUniqueParents(const CTransaction& tx);
 
+    /** If a transaction is an orphan resolution candidate, treat the unique_parents as announced by this peer; add them
```

Comment:
they are added as new invs IFF they are deemed good candidates for resolution

Reply:
thanks, edited

Reply:
Thanks for the update even though on second reading it was already ok :)

Reply:
🫡 I am p easy-going